store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/key/core/impl/bytedEntity/ByteEntityAccumuloElementConverterTest.java

```diff
@@ -72,14 +72,15 @@ public void shouldSerialiseWithHistoricColumnQualifier() throws Exception {
                 put(AccumuloPropertyNames.COLUMN_QUALIFIER_4, Integer.MIN_VALUE);
             }
         };
-        byte[] historicColumnQualifierBytes = {4, 1, 0, 0, 0, 4, -1, -1, -1, 127, 4, 3, 0, 0, 0, 4, 0, 0, 0, -128};
+        byte[] historicColumnQualifierBytes = {4, 1, 0, 0, 0, 4, -1, -1, -1, 127, 4, 3, 0, 0, 0, 4, 0, 0, 0, -128, 0};
```

Comment:
Breaking change

Reply:
The only reason a 0 is added to the end of the byte array is because I changed the test to use "schemaWithVisibilities", and therefore, the 0 is used to serialise that extra property. Can confirm that this works with prior versions of Gaffer. To guarantee this though, perhaps this test should be duplicated, once with and once without visibilities in current Gaffer, then this branch can just prove that it passes both?

Reply:
Me and @GCHQDev404 have decided that the best way forward is to leave this test alone and add a second test that uses the visibility schema and merge that into Gaffer first. Then this branch can just show that it passes both tests. See #2488